src/User.hs

```diff
@@ -1,6 +1,9 @@
 {-# LANGUAGE DeriveGeneric #-}
 {-# LANGUAGE ImportQualifiedPost #-}
 {-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE DuplicateRecordFields #-}
```

Comment:
DuplicateRecordFields mit NoFieldSelectors noch nötig?

Reply:
Yes, `DuplicateRecordFields` is still necessary even though we are using `NoFieldSelectors`.